test/device_map_dialog_test.rb

```diff
@@ -0,0 +1,69 @@
+require_relative "test_helper"
+
+require "bootloader/device_map_dialog"
+
+describe Bootloader::DeviceMapDialog do
+  # just simple tests to avoid typos as logic is not so easy to test
+  describe "#run" do
+    before do
+      Yast.import "BootStorage"
+
+      device_map = Bootloader::DeviceMap.new(
+        "/dev/sda" => "hd0",
+        "/dev/sdb" => "hd1"
+      )
+      allow(Yast::BootStorage).to receive(:DisksOrder)
+        .and_return(device_map.disks_order)
+    end
+
+    def mock_ui_events(*events)
```

Comment:
Well, I guess that before adding something to the rspec helpers we should agree some general approach to UI testing in YaST (we have a very short "tradition" in testing UI).